bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/package.scala

```diff
@@ -1,4 +1,38 @@
 package edu.gemini.phase2.skeleton
 
+import edu.gemini.model.p1.immutable.{Band, Observation, Proposal, TimeAmount}
+
+import scalaz._
+import Scalaz._
+
 package object factory {
+  // Filter to only the enabled observations by appropriate band.
+  def enabledObs(proposal: Proposal): List[Observation] = {
+    val band = extractBand(proposal)
+    proposal.observations.filter(obs => obs.band == band && obs.enabled)
+  }
+
+  // If there is an itac acceptance, then use its band assignment.  Otherwise
+  // just figure we will use the "normal" band 1/2 observations.
+  def extractBand(proposal: Proposal): Band =
+    (for {
+      itac   <- proposal.proposalClass.itac
+      accept <- itac.decision.right.toOption
+    } yield accept.band).getOrElse(1) match {
+      case 3 => Band.BAND_3
+      case _ => Band.BAND_1_2
```

Comment:
Glad to hear it 😸, since I extracted this directly from `Phase1FolderFactory`, assuming that that code was doing the correct thing.